packages/destination-actions/src/destinations/amplitude/identifyUser/index.ts

```diff
@@ -242,6 +242,10 @@ const action: ActionDefinition<Settings, Payload> = {
     let options
     const properties = rest as AmplitudeEvent
 
+    if (properties.platform) {
+      properties.platform = properties.platform.replace('ios', 'iOS').replace('android', 'Android')
```

Comment:
We may want to be extra flexible and accept case insensitive input, while replacing with case-sensitive versions that Amplitude expects.

```suggestion
      properties.platform = properties.platform.replace(/ios/i, 'iOS').replace(/android/i, 'Android')
```